rmw_cyclonedds_cpp/CMakeLists.txt

```diff
@@ -37,14 +37,20 @@ if(NOT CycloneDDS_FOUND)
   return()
 endif()
 
+if("$ENV{ROS_DISTRO}" STRLESS "foxy")
+  set(rosidl_runtime_pkgs rosidl_generator_c)
```

Comment:
Nitpick: maybe us singular variable name since it only contains a single package name. Also the variable doesn't contain rosidl all runtime packages but only the one for C.